### DISCUSSION TITLE
[Lesson 7 -- Foundry Fund Me]  Unable to access 'https://github.com/foundry-rs/forge-std/'

### DISCUSSION THREAD

**Discussion by suryaPro3909** [Cyfrin/foundry-full-course-cu#959] [Q&A]
I was creating a foundry project but didn't got forge-std library , then again entered "forge init --force " but  it was showing the following error.

```
surya@LAPTOP-TOB9PJ92:~/blockchain/FundMe$ forge init --force
Target directory is not empty, but `--force` was specified
Initializing /home/surya/blockchain/FundMe...
Installing forge-std in /home/surya/blockchain/FundMe/lib/forge-std (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
Cloning into '/home/surya/blockchain/FundMe/lib/forge-std'...
fatal: unable to access 'https://github.com/foundry-rs/forge-std/': Could not resolve host: github.com
fatal: clone of 'https://github.com/foundry-rs/forge-std' into submodule path '/home/surya/blockchain/FundMe/lib/forge-std' failed
Error: 
git submodule exited with code 128

``` 
My internet connection is good.
Also tried many commands like

```
git config --global --unset http.proxy 
git config --global --unset https.proxy
```
also cleared DNS cache but did not get it resolved.
Please resolve my query .

**Comment by alfheimrShiven** [comment]
Hey @suryaPro3909 I'm guessing this can be due to some access issue. Are you logged into your Github account locally? You can check so by running the cmd: `git config user.name`
Let me know if it returns your username or null. If null, maybe log into Github through the terminal and try initializing the project again.

**Comment by Mohammad-Ahadinejad** [comment]
Hey @suryaPro3909 . Did you find the answer? I have the same problem unfortunately.

**Reply by JASH-PATEL-6566** [reply]
I have one solution, In my system the problem occur due to DNS issue

so I simply run command  ```sudo nano /etc/resolv.conf``` and change nameserver xx.xx.xx.xx to `8.8.8.8`

I am good to go now, I hope this is helpful to you also.

**Reply by dpzengheng** [reply]
I have one solution, i run command  sudo vi /etc/hosts and add to last row  like '140.82.112.4 github.com'

**Reply by EngrPips** [reply]
Interesting.

**Comment by cryptjohel** [comment]
I just encountered the same issue here how to solve it. In that directory created install ``` forge install foundry-rs/forge-std ```
and after that delete the existing forge modules that u have activated with forge init and then say ``` forge init --force```
